beetmoverscript/script.py

```diff
@@ -192,6 +192,7 @@
                 context.config.get('zip_max_file_size_in_mb', DEFAULT_ZIP_MAX_FILE_SIZE_IN_MB),
                 artifact_map=context.task['payload'].get('artifactMap')
             )
+            await move_beets(context, context.artifacts_to_beetmove, artifact_map=context.task['payload']['artifactMap'])
```

Comment:
Facepalm 🤦‍♂ 😳  Sorry.

Reply:
I'm sorry too. I should have caught this in review too.